wasm/spectests/util.go

```diff
@@ -0,0 +1,200 @@
+package spectests
```

Comment:
note this is just moved from spec_test.go so we can reuse it in wazeroir/compier_test.go